packages/services/service-common/src/trpc.ts

```diff
@@ -28,7 +28,8 @@ export const handleTRPCError = experimental_standaloneMiddleware<{
   ctx: {
     req: FastifyRequest;
   };
-  input: unknown;
+  // eslint-disable-next-line @typescript-eslint/no-empty-object-type
+  input: {};
```

Comment:
```suggestion
  input: unknown;
```

Reply:
Do we have this `unknown` on `main` already? I'm not sure changing this didn't cause some type errors in GraphQL related types.